sources/packages/api/src/services/disbursement-schedule-service/disbursement-schedule.models.ts

```diff
@@ -19,3 +22,8 @@ export enum EnrollmentPeriod {
   Current = "current",
   Upcoming = "upcoming",
 }
+
+export interface DisbursementScheduleWithStopFullTimeBCFundingStatus
+  extends DisbursementSchedule {
+  stopFullTimeBCFundingStatus: number;
```

Comment:
stopFullTimeBCFunding**Status**??

Reply:
@ann-aot, if this is about the variable naming, it is not keeping the status (IMO), it defines if the exception is present or not. My recommendation[ in the other comment ](https://github.com/bcgov/SIMS/pull/1265#discussion_r894065676) was to change it to a boolean and name it `stopFullTimeBCFunding`.